cypress/integration/utils/helpers.ts

```diff
@@ -158,3 +148,38 @@ export function newDevice(
 export function newShouldArgs(chainer: string, value?: any): ShouldArgs {
   return { chainer, value };
 }
+
+export function newExpectationWithSelector(
+  description: string,
+  selector: string,
+  selectOption: number,
+  shouldArgs: ShouldArgs,
+): Expectation {
+  return { description, selector, shouldArgs, selectOption };
+}
+export function newExpectationWithVisitPage(
```

Comment:
Super small nit, but functions should have an empty line after them
```suggestion
export function newExpectationWithSelector(
  description: string,
  selector: string,
  selectOption: number,
  shouldArgs: ShouldArgs,
): Expectation {
  return { description, selector, shouldArgs, selectOption };
}

export function newExpectationWithVisitPage(
```